src/components/GeneralInfo/BiosCard/BiosCard.test.js

```diff
@@ -1,11 +1,9 @@
 /* eslint-disable camelcase */
 import React from 'react';
-import { render } from 'enzyme';
-import toJson from 'enzyme-to-json';
 import BiosCard from './BiosCard';
 import configureStore from 'redux-mock-store';
 import { biosTest } from '../../../__mocks__/selectors';
-import { mountWithRouter } from '../../../Utilities/TestingUtilities';
+import { renderWithRouter } from '../../../Utilities/TestingUtilities';
```

Comment:
Maybe it would be better to move the `render` out of the `mount/renderWithRouter` (and maybe even call it `TestWrapper`) and import the `render` directly here. 

I think that it has the advantage of it being visible right away what test framework is used. Another thing is, that `renderWithRouter` adds another layer, and while I doubt it, it has the potential of introducing unwanted behaviour. 

Having a `TestWrapper` component might be a pattern we could apply to more applications. It would basically be the same as any `App.js` with all the necessary provider components.

Reply:
Yeah, sounds very similar to what happened to `mountWithContext` from cypress tests. I'll rewrite this part now